lib/types.ts

```diff
@@ -827,12 +829,13 @@ export type FlexCarousel = {
  *
  * - [Box](https://developers.line.me/en/reference/messaging-api/#box)
  * - [Button](https://developers.line.me/en/reference/messaging-api/#button)
- * - [Filler](https://developers.line.me/en/reference/messaging-api/#filler)
- * - [Icon](https://developers.line.me/en/reference/messaging-api/#icon)
  * - [Image](https://developers.line.me/en/reference/messaging-api/#f-image)
- * - [Separator](https://developers.line.me/en/reference/messaging-api/#separator)
- * - [Spacer](https://developers.line.me/en/reference/messaging-api/#spacer)
+ * - [Icon](https://developers.line.me/en/reference/messaging-api/#icon)
```

Comment:
https://developers.line.biz/en/reference/messaging-api/#icon
Change `line.me`(old) domain to `line.biz` (new) domain.